vshard/storage/init.lua

```diff
@@ -1808,6 +1809,9 @@ local function bucket_test_gc(bids)
     local _bucket = box.space._bucket
     local ref, bucket, status
     for i, bid in ipairs(bids) do
+        if M.rebalancer_transfering_buckets[bid] then
```

Comment:
Sounded like a good idea, but then I saw that it won't work as expected. In `recovery_step_by_type()` there is this place:
```
        if not bucket or not bucket_is_transfer_in_progress(bucket) then
            goto continue
        end
```
If a bucket is `ACTIVE` but has the flag (its sending might need time to start if there are concurrent map-calls, or it might be just not yet visible due to mvcc), then we will try to "recover" it. Shouldn't do that.

Reply:
Agree, we should not. Even though it seems to me, that we have to process that case individually and still use `M.rebalancer_transfering_buckets` as a characteristic of `bucket_transfer_in_progress`, I don't mind if we stay as is, anyway it has little connection with the current patchset.